src/Dropdown.svelte

```diff
@@ -91,10 +91,13 @@
 
     toggle(e);
   }
+
   onDestroy(() => {
-    ['click', 'touchstart', 'keyup'].forEach((event) =>
-      document.removeEventListener(event, handleDocumentClick, true)
-    );
+    if (typeof document !== 'undefined') {
```

Comment:
Would `if (document !== undefined)` or `if (document)` suffice here?

Reply:
I just copypasted the check from line 57
![](https://i.shx.gg/i8gDBk.png)
doesn't look like it would.